project/AppDependencies.scala

```diff
@@ -4,30 +4,31 @@ import sbt._
 
 object AppDependencies {
 
-  val compile = Seq(
-    "uk.gov.hmrc.mongo"      %% "hmrc-mongo-play-28"         % "0.68.0",
-    "uk.gov.hmrc"            %% "play-frontend-hmrc"         % "3.21.0-play-28",
+  val hmrcMongoVersion = "0.71.0"
```

Comment:
I think the same is here we agreed to update till 0.68 to avoid any errors but I am happy to give it a try